pypeit/find_objects.py

```diff
@@ -1188,8 +1188,8 @@ def calculate_flexure(self, global_sky):
         sl_ref = self.par['calibrations']['flatfield']['slit_illum_ref_idx']
         box_rad = self.par['reduce']['extraction']['boxcar_radius']
         trace_spat = 0.5 * (self.slits_left + self.slits_right)
-        # Load archival sky spectrum for absolute correction
-        sky_spectrum, sky_fwhm_pix = flexure.get_archive_spectrum(self.par['flexure']['spectrum'])
+        # # Load archival sky spectrum for absolute correction
```

Comment:
These two commented lines can be safely removed  :-)

Reply:
Done